<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Ravi.local" ignored="0" name="Default test(failed)" tests="3" failures="2" timestamp="08 Jul 2020 17:09:44 GMT" time="26.915" errors="0">
  <testcase name="takeScreenshot1" time="0.011" classname="com.test.program.ScreenshotFailedTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.test.program.ScreenshotFailedTest.takeScreenshot1(ScreenshotFailedTest.java:23)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- takeScreenshot1 -->
  <testcase name="takeScreenshot2" time="0.002" classname="com.test.program.ScreenshotFailedTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.test.program.ScreenshotFailedTest.takeScreenshot2(ScreenshotFailedTest.java:27)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- takeScreenshot2 -->
  <testcase name="takeScreenshot" time="0.001" classname="com.test.program.ScreenshotFailedTest"/>
</testsuite> <!-- Default test(failed) -->
